app/Repositories/ProfileRepository.php

```diff
@@ -115,6 +115,29 @@ public function sortGroupsByDisplayOrder($grouped, $groups)
             ->toArray();
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function getProfilesByGroupOrderPiped($site_id, $groups)
```

Comment:
Think "piped" should be removed or renamed.

Reply:
Does anyone have any suggestions for a better name? I also think it's wordy, but dropping "Piped" makes it sound too similar to both "getProfilesByGroup" and "sortGroupsByDisplayOrder". There honestly is probably a better way to name this, but I feel like I've been looking at it for too long and I'm too close to it to come up with an effective one.

Reply:
If "getProfilesByGroupOrder" is what the function does then thats an OK name, regardless of how close it is to others.